api/buildstatus/buildstatus_handler.go

```diff
@@ -0,0 +1,151 @@
+package buildstatus
+
+import (
+	"bytes"
+	"fmt"
+	"html/template"
+	"log"
+	"sort"
+	"strings"
+
+	"github.com/equinor/radix-api/api/utils"
+	"github.com/equinor/radix-api/models"
+	v1 "github.com/equinor/radix-operator/pkg/apis/radix/v1"
+	"github.com/marstr/guid"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const BUILD_STATUS_SVG_RELATIVE_PATH = "../../badges/build-status.svg"
+
+type BuildStatusHandler struct {
+	accounts models.Accounts
+}
+
+func Init(accounts models.Accounts) BuildStatusHandler {
+	return BuildStatusHandler{accounts: accounts}
+}
+
+// GetBuildStatusForApplication Gets a list of build status for environments
+func (handler BuildStatusHandler) GetBuildStatusForApplication(appName, env string) (*[]byte, error) {
+	var output []byte
+
+	// Get latest RJ
+	serviceAccount := handler.getServiceAccount()
+	namespace := fmt.Sprintf("%s-app", appName)
+
+	// Get list of Jobs in the namespace
+	radixJobs, err := serviceAccount.RadixClient.RadixV1().RadixJobs(namespace).List(metav1.ListOptions{})
+
+	if err != nil {
+		return nil, err
+	}
+
+	latestBuildDeployJob, err := getLatestBuildJobToEnvironment(radixJobs.Items, env)
+
+	if err != nil {
+		return nil, utils.NotFoundError(err.Error())
+	}
+
+	buildStatus := latestBuildDeployJob.Status.Condition
+
+	if buildStatus == "Succeeded" {
+		output = append(output, *writePassingSvg("build")...)
+	} else if buildStatus == "Failed" {
+		output = append(output, *writeFailingSvg("build")...)
+	} else {
+		output = append(output, *writeUnknownSvg("build")...)
+	}
```

Comment:
The list of statuses are defined as constants in radix-operator.
```
	JobWaiting   RadixJobCondition = "Waiting"
	JobRunning   RadixJobCondition = "Running"
	JobSucceeded RadixJobCondition = "Succeeded"
	JobFailed    RadixJobCondition = "Failed"
	JobStopped   RadixJobCondition = "Stopped"
```
Use these in the comparison instead of a string.
`if buildStatus == v1.JobSucceeded`

Shouldn't the badge reflect the other statuses as well? Now it only handles Succeeded and Failed. The others are unknown. I suggest we show all the different statuses to the user. Pick a suitable color scheme for each of them.